# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/RemoteDecoderManagerParent.h
# Commit: 557f55a90c8a
# Full Hash: 557f55a90c8aa215b7046fadf7a17436baa93713
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-06-27 09:34:48
# Regressor Bug: 1561178
# File Overlap Count: 1
# Description:
#   Bug 1561178 - Make passing a TextureFactoryIdentifier optional, and pass one whenever we can. r=jya
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D35965
# ==============================================================================

diff -r e1521d8fbe90 -r 557f55a90c8a dom/media/ipc/RemoteDecoderManagerParent.h
--- a/dom/media/ipc/RemoteDecoderManagerParent.h	Thu Jun 27 01:28:47 2019 +0000
+++ b/dom/media/ipc/RemoteDecoderManagerParent.h	Thu Jun 27 01:28:56 2019 +0000
@@ -35,9 +35,9 @@
   PRemoteDecoderParent* AllocPRemoteDecoderParent(
       const RemoteDecoderInfoIPDL& aRemoteDecoderInfo,
       const CreateDecoderParams::OptionSet& aOptions,
-      const layers::TextureFactoryIdentifier& aIdentifier, bool* aSuccess,
-      nsCString* aBlacklistedD3D11Driver, nsCString* aBlacklistedD3D9Driver,
-      nsCString* aErrorDescription);
+      const Maybe<layers::TextureFactoryIdentifier>& aIdentifier,
+      bool* aSuccess, nsCString* aBlacklistedD3D11Driver,
+      nsCString* aBlacklistedD3D9Driver, nsCString* aErrorDescription);
   bool DeallocPRemoteDecoderParent(PRemoteDecoderParent* actor);
 
   mozilla::ipc::IPCResult RecvReadback(const SurfaceDescriptorGPUVideo& aSD,